pyhf/utils.py

```diff
@@ -157,19 +164,50 @@ def runOnePoint(muTest, data, pdf, init_pars=None, par_bounds=None):
     asimov_mu = 0.0
     asimov_data = generate_asimov_data(asimov_mu, data, pdf, init_pars, par_bounds)
 
-    qmu_v = tensorlib.clip(qmu(muTest, data, pdf, init_pars, par_bounds), 0, max=None)
+    qmu_v = tensorlib.clip(qmu(mu_test, data, pdf, init_pars, par_bounds), 0, max=None)
     sqrtqmu_v = tensorlib.sqrt(qmu_v)
 
     qmuA_v = tensorlib.clip(
-        qmu(muTest, asimov_data, pdf, init_pars, par_bounds), 0, max=None
+        qmu(mu_test, asimov_data, pdf, init_pars, par_bounds), 0, max=None
     )
     sqrtqmuA_v = tensorlib.sqrt(qmuA_v)
 
     CLsb, CLb, CLs = pvals_from_teststat(sqrtqmu_v, sqrtqmuA_v)
-
-    CLs_exp = []
-    for nsigma in [-2, -1, 0, 1, 2]:
-        sqrtqmu_v_sigma = sqrtqmuA_v - nsigma
-        CLs_exp.append(pvals_from_teststat(sqrtqmu_v_sigma, sqrtqmuA_v)[-1])
-    CLs_exp = tensorlib.astensor(CLs_exp)
-    return qmu_v, qmuA_v, CLsb, CLb, CLs, CLs_exp
+    _returns = CLs
+    if kwargs:
+        _returns = [CLs]
+        if 'observed_set' in kwargs:
+            observed_set = kwargs['observed_set']
+        else:
+            observed_set = False
```

Comment:
could just be `kwargs.get('observed_set')`. Not sure we need to assign to local vars..

```
if kwargs.get('test_statistics'): _returns.append([qmu_v, qmuA_v])
```

Reply:
Good point

Reply:
Changed in https://github.com/diana-hep/pyhf/pull/341/commits/490b22da1b8d2c6849523dcbfe340a211ac4dfd5